pcsx2/DEV9/Win32/tap-win32.cpp

```diff
@@ -355,7 +355,7 @@ bool TAPGetWin32Adapter(const std::string& name, PIP_ADAPTER_ADDRESSES adapter,
 
 	//We must be bridged
 	Console.WriteLn("DEV9: Current adapter is probably bridged");
-	Console.WriteLn(L"DEV9: Adapter Display name: %s", pAdapterInfo->FriendlyName);
+	Console.WriteLn("DEV9: Adapter Display name: %s", pAdapterInfo->FriendlyName);
```

Comment:
`pAdapterInfo->FriendlyName` is a wide string

I think wx let me get away with using `%s` instead of `%ls`

Reply:
%ls is potentially problematic, as _scprintf with a %ls string that has any codepoints above 255 will return -1, and blow up the string conversion. I'll just convert them to UTF-8 first.